src/adapters/IDB.js

```diff
@@ -427,7 +427,8 @@ function transactionProxy(store, preloaded = []) {
     },
 
     update(record) {
-      store.put(record);
+      // We need to put undefined there so that Safari and IE are happy.
+      store.put(record, undefined);
```

Comment:
This is the case for Chrome and Firefox but apparently Safari and IE checks for the parameters length rather than if the value is set to `undefined`: http://stackoverflow.com/a/19318034/186202